app/views/steps/completion/what_next/show.en.html.erb

```diff
@@ -13,7 +13,9 @@
       </strong>
     </div>
 
-    <section class="numbered-list">
+    <!-- TODO: remove the `section` wrapper when updating this view to new design system
```

Comment:
is this left intentionally here?

Reply:
Yes, I added that comment to not forget when we migrate the completion page, to remove the wrapper around the list (section) and to move the `app-list--icon-steps` class into the list.
I didn't want to do the migration of this page as part of this PR.